pvlib/tests/test_conftest.py

```diff
@@ -19,3 +20,25 @@ def test_fail_on_pvlib_version_pass():
 @fail_on_pvlib_version('100000.0')
 def test_fail_on_pvlib_version_fail_in_test():
     raise Exception
+
+
+# set up to test passing arguments to conftest.fail_on_pvlib_version
+@pytest.fixture()
+def some_data():
+    return "some data"
+
+
+def alt_func(*args):
+    return args
+
+
+deprec_func = deprecated('350.8', alternative='alt_func',
+                         name='deprec_func', removal='350.9')(alt_func)
+
+
+@fail_on_pvlib_version('350.9')
+def test_deprecated_09(some_data):
```

Comment:
nit: shouldn't the function name be something like `test_fail_on_pvlib_version_args_kwargs`?